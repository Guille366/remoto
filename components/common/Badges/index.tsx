import React from "react";

const Badges = ({
  newOpening,
  hotOpening,
}: {
  newOpening: boolean;
  hotOpening: boolean;
}) => {
  return (
    <div className="opacity-80 font-bold font-code absolute -mt-1 top-0 left-0 p-2 pt-1">
      <span className="text-red-600 relative">
        <small className="absolute animate-bounce pt-1">
          {newOpening && "Nova!"}
        </small>
      </span>
      <span
        className={`text-yellow-500 ${
          newOpening ? "left-12" : "left-0"
        } relative`}
      >
        <small className="absolute animate-bounce pt-1">
          {hotOpening && "Hot!"}
        </small>
      </span>
    </div>
  );
};

export default Badges;
