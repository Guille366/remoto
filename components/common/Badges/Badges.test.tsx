import { render } from "@testing-library/react";
import Badges from ".";

describe('Loading', () => {
  it('renders properly', () => {
    const { container } = render(<Badges hotOpening newOpening />);

    expect(container.firstChild).toMatchSnapshot();
  });
});