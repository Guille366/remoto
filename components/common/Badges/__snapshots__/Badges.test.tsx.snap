// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders properly 1`] = `
<div
  class="opacity-80 font-bold font-code absolute -mt-1 top-0 left-0 p-2 pt-1"
>
  <span
    class="text-red-700 relative"
  >
    <small
      class="absolute animate-bounce pt-1"
    >
      Nova!
    </small>
  </span>
  <span
    class="text-yellow-600 left-12 relative"
  >
    <small
      class="absolute animate-bounce pt-1"
    >
      Hot!
    </small>
  </span>
</div>
`;
